### DISCUSSION TITLE
Primefaces datatable not rendering after upgrade from Primefaces 8.0 to 11.0.0

### DISCUSSION THREAD

**Discussion by mbecks3** [primefaces/community#1074] [PrimeFaces] [answered]
I'm having an issue with Primefaces datatable rendering after updating from 8.0 to 11.0.0.  
JDK 17, Tomcat  10.1, Primefaces 11, myfaces 4.0.1.
When clicking my commandButton, the datatable on numerous pages does not render results.  Upon entering the page, the datatable is rendered with the emptyMessage="No Data Systems".
I am able to show that my actionListener="#{activityManagedBean.searchAction}" returns a list of objects that should populate my dataTable.  I'll post all relevant code examples below, and the following code is/was working on PF 8.  I'll post all relevant files, and this was the closest issue I was able to find on this [(https://forum.primefaces.org/viewtopic.php?t=72508)].  Thank you for the assistance.
<img width="956" alt="afterButtonClick" src="https://github.com/primefaces/community/assets/158097881/374f8b20-6957-400d-a439-f05559f68cd4">
<img width="955" alt="beforeButtonClick" src="https://github.com/primefaces/community/assets/158097881/cd103b35-8e03-4d21-b0a6-aca3a8f01fd3">
[dataSystems.txt](https://github.com/primefaces/community/files/14117256/dataSystems.txt)
[search.txt](https://github.com/primefaces/community/files/14117257/search.txt)

`search.xhtml:`

```
<h:panelGroup>
    <p:spacer width="10"/>
    <p:commandButton id="searchCB" value="Search" partialSubmit="true" process="mainForm:searchCriteriaOP" actionListener="# 
                           {activityManagedBean.searchAction}" oncomplete="$('#mainForm').show()" update="mainForm"/>
    <p:commandButton id="clearCB" value="Clear" immediate="true" partialSubmit="true" process="@this" actionListener="# 
                           {activityManagedBean.clearSearchAction}" oncomplete="$('#mainForm').show()" resetValues="true" 
                           update="mainForm"/>
</h:panelGroup>
```

`dataSystems.xhtml:`

```
<p:dataTable id="dataSystemsData" widgetVar="dataSystemsDataVar"
				styleClass="maxWidth" var="row" tableStyle="table-layout:auto"
				value="#{activityManagedBean.dataSystemList}"
				selection="#{activityManagedBean.selectedDataSystem}"
				resizableColumns="true" rowKey="#{row.dsd}" reflow="true"
				rows="100" paginator="true" paginatorAlwaysVisible="true"
				paginatorPosition="top" rowsPerPageTemplate="10,25,50,100"
				emptyMessage="No Data Systems" >
    <p:ajax event="rowSelectRadio" immediate="true" update="mainForm:dataSystemsTB" global="false" />
    <p:column styleClass="align-center" selectionMode="single" width="5%" rendered="#{User.isSysAdmin}"/>						
    <p:column styleClass="align-center" headerText="DSD" width="10%" sortBy="#{row.dsd}">
        <p:commandLink onclick="window.open('/CDRS/page/viewDataSystem.xhtml?dsd=#{row.dsd}','_blank')">
	    <h:outputText styleClass="font-12" value="#{row.dsd}" />
	</p:commandLink>															
    </p:column>
    <p:column styleClass="align-center" headerText="Name" width="45%" sortBy="#{row.name}">
        <h:outputText value="#{row.name}" />
    </p:column>
    <p:column styleClass="align-center" headerText="Acronym" width="20%" sortBy="#{row.acronym}">
        <h:outputText value="#{row.acronym}" />
    </p:column>
    <p:column styleClass="align-center" headerText="ITIPS ID" width="20%" sortBy="#{row.itipsId}">
         <h:outputText value="#{row.itipsId}" />
    </p:column>																												
</p:dataTable>
```

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Comment by TevfikFurkanAksu** [comment, marked as answer]
change reflow attiribute as false.
